File: langium/Java/LangiumLexerBase.java
import org.antlr.v4.runtime.CharStream;
import org.antlr.v4.runtime.Lexer;

public abstract class LangiumLexerBase extends Lexer {

  protected LangiumLexerBase(CharStream input) {
    super(input);
  }

  protected boolean NoSlash() {
    return this._input.LA(1) != '/';
  }
}

File: langium/CSharp/LangiumLexerBase.cs
using System;
using System.Collections.Generic;
using System.IO;
using Antlr4.Runtime;

public abstract class LangiumLexerBase : Lexer
{
	public LangiumLexerBase(ICharStream input)
			: base(input)
	{
	}

	public LangiumLexerBase(ICharStream input, TextWriter output, TextWriter errorOutput)
			: base(input, output, errorOutput)
	{
	}

	public bool NoSlash() => this.InputStream.LA(1) != '/';
}
